datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -690,7 +705,8 @@ private DatastreamTask getDatastreamTask(String taskName) {
     if (isDatastreamUpdate || !addedTasks.isEmpty() || !removedTasks.isEmpty()) {
       // Populate the event producers before calling the connector with the list of tasks.
       addedTasks.stream().filter(t -> t.getEventProducer() == null).forEach(this::initializeTask);
-      return submitAssignment(connectorType, assignment, isDatastreamUpdate, connector, removedTasks, retryAndSaveError);
+      return submitAssignment(connectorType, assignment, isDatastreamUpdate, connector, removedTasks,
```

Comment:
I don't see any change here.

Reply:
Looks like I split this line into two lines by mistake, will fix it.